cabot/statuscheckapp/models/base.py

```diff
@@ -0,0 +1,137 @@
+from celery.utils.log import get_task_logger
+from django.db import models
+import json
+import time
+from cabot.cabotapp.models import CHECK_TYPES, StatusCheck, StatusCheckResult
+
+logger = get_task_logger(__name__)
+
+
+class MetricsInstanceBase(models.Model):
+    class Meta:
+        app_label = 'statuscheckapp'
+
+    name = models.CharField(
+        unique=True,
+        max_length=30,
+        help_text='Unique name for the data source',
+    )
+
+
+class MetricsStatusCheckBase(StatusCheck):
+    class Meta:
+        app_label = 'statuscheckapp'
+
+    metrics_instance = models.ForeignKey(MetricsInstanceBase)
+    check_type = models.CharField(
+        choices=CHECK_TYPES,
+        max_length=30,
+    )
+    value = models.FloatField(
```

Comment:
Think this would be easiest to do as 3 separate fields:
`critical_value: value1
error_value: value2
warning_value: value3`
All of them could be nullable

Then when running the check we check failure against critical then error (if critical passes) then warning (if error passes) and set the `importance` accordingly. 

This way we shouldn't have to change any existing alerting code and don't have to deal with arbitrary #s of values